util/config/config.go

```diff
@@ -539,11 +575,13 @@ func (config *Configuration) incrementPort() {
 	allPorts := []uint16{
 		config.NodePort,
 		config.HttpWsPort,
+		config.HttpWssPort,
+		config.HttpsJsonPort,
```

Comment:
Suggest adjust slice order, for `base+offset` coincident with port's actual purpose.
```
base_port + 1 = NodePort
base_port + 2 = WsPort
base_port + 3 = JsonPort
base_port + 4 = WssPort
base_port + 5 = sJsonPort
```

Reply:
done